testhelpers/image_privileges.go

```diff
@@ -24,11 +24,11 @@ func NewImagePrivileges(imageName string) (priv ImagePrivileges) {
 	return
 }
 
-// Based of the registry API specification https://docs.docker.com/registry/spec/api/
-// This method returns the image name for path value that matches requests to blobs, manifests or tags
+// extractImageName returns the image name for path value that matches requests to blobs, manifests or tags
 // For examples:
 // extractImageName("v2/foo.bar/blobs/") returns "foo.bar"
 // extractImageName("v2/foo/bar/manifests/") returns "foo/bar"
+// Based of the registry API specification https://docs.docker.com/registry/spec/api/
```

Comment:
```suggestion
// Based on the Docker registry API specification: https://docs.docker.com/registry/spec/api/
```

Reply:
Fixed in the following [commit](https://github.com/buildpacks/imgutil/pull/126/commits/1a4373ac85d225b6bd3771c3ccadae2a239cf6db)